src/pages/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+import './login.css';
+
+function LoginPage() {
+  return (
+    <div className="loginpage">
+      <h1>Login</h1>
+      <div className="centered">
+        <form className="myform" action="" method="POST">
+          <input className="formitem" type="text" id="name" name="name" placeholder="Name" required minLength="2" maxLength="30" />
+          <input className="formitem" type="password" id="password" name="password" placeholder="Password" required />
+          <div className="button-wrapper">
+            &nbsp;
+            <button className="sbutton" type="submit">Submit</button>
+            &nbsp;
+          </div>
```

Comment:
I think we have a Login route on the header, would you want to try and use that route and change the onclick doLogin to the submit button on this page?

Reply:
I don't understand. Do you mean because of the DRY principle? The header link is not definitive, it's there to try and debug. I think the onSubmit form's event handler does the job ok, still, it's open for change because I still need to respond to login success or failure.